srcpkgs/VeraCrypt/template

```diff
@@ -14,8 +14,8 @@ short_desc="Disk encryption with strong security based on TrueCrypt"
 maintainer="Gustavo Heinz <gustavoheinz95@gmail.com>"
 license="Apache-2.0, TrueCrypt 3.0"
 homepage="https://www.veracrypt.fr"
-distfiles="${SOURCEFORGE_SITE}/veracrypt/VeraCrypt_${_realversion}_Source.zip"
-checksum=229de81b2478cfa5fa73e74e60798a298cd616e9852b9f47b484c80bc2a2c259
+distfiles="https://launchpad.net/veracrypt/trunk/1.24-update2/+download/VeraCrypt_${_realversion}_Source.tar.bz2"
```

Comment:
why not download from upstream's cgit?
https://www.veracrypt.fr/code/VeraCrypt/tag/?h=VeraCrypt_1.24-Update2
in case the tarballs change, check out the manual for archive content hashing (`@`)

Reply:
That is the download link the upstream provides.